src/containers/App/reducer.js

```diff
@@ -16,7 +13,9 @@ import {
   UPLOAD_PROGRESS,
   UPLOAD_SUCCESS,
   UPLOAD_FAILURE,
-} from './constants';
+  SET_SEARCH_QUERY,
+  RESET_SEARCH_QUERY,
+} from 'containers/App/constants';
```

Comment:
I'd use `from './constants';` here to signify that it's coming from the same folder